src/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+find_package(Freetype REQUIRED)
+find_package(glad CONFIG REQUIRED)
+find_package(glfw3 CONFIG REQUIRED)
+find_package(glm CONFIG REQUIRED)
+find_package(libpng CONFIG REQUIRED)
+find_package(LibSndFile CONFIG REQUIRED)
+find_package(NovelRT CONFIG REQUIRED)
+find_package(OpenAL CONFIG REQUIRED)
+find_package(spdlog CONFIG REQUIRED)
```

Comment:
Surely we should just have to `find_package NovelRT` here? As opposed to all of them?

Reply:
We get errors like:
```
CMake Error at artifacts/vcpkg/scripts/buildsystems/vcpkg.cmake:188 (_add_executable):
  Target "TouhouNovelRT" links to target "glad::glad" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  src/CMakeLists.txt:13 (add_executable)
```

I'd guess I just don't have the targets quite right and it's something I'll have to investigate more to fix.

Reply:
Do you think we could get this fixed before merging? Or nah?

Reply:
Discussed on Discord. Will resolve at a later date.